protocol/common/message.go

```diff
@@ -170,6 +170,8 @@ type Message struct {
 	AudioPath string `json:"audioPath,omitempty"`
 	// ImageLocalURL is the local url of the image
 	ImageLocalURL string `json:"imageLocalUrl,omitempty"`
+	// AlbumCount for a collage of images
+	AlbumCount int `json:"albumCount,omitempty"`
```

Comment:
Wdyt about the name `AlbumSize`? Sounds a bit clearer to me. Also, I'd adjust the comment line to be "AlbumCount is a number of images in a collage"

Reply:
Personnally .. I prefer `count` I think it refers to integer .. Will edit the comment though

Reply:
It sounds a bit of "bad English" to me. "Album count" can be incorrectly understood as the number of albums. Maybe `AlbumPhotosCount`?

Reply:
That's exactly how I felt about the word `count`. I had to re-read many times the name `AlbumCount` until I was sure it wasn't a count of albums, but the count of messages of type *image*.

Reply:
Well if you put it that way, then I guess it's a bad name
Okie dokie
Renaming

Reply:
Done